src/figuresClasses.ts

```diff
@@ -1,11 +1,108 @@
-export interface Figure {}
+type ColorType = 'red' | 'green' | 'blue';
 
-export class Triangle implements Figure {}
+enum ShapeEnum {
+  TRIANGLE = 'triangle',
+  CIRCLE = 'circle',
+  RECTANGLE = 'rectangle',
+}
```

Comment:
[CODE STYLE] - Enum constant names usually start with a capital letter. Please adjust the `ShapeEnum` to follow this convention. For example, `TRIANGLE` should be `Triangle`.